### PR TITLE
Add a few helpers for quickly finding task_structs and file structs within them

### PR DESCRIPTION
**Pull request by sdimitro**
There are 2 commits here. The first one is a side-fix for an f-string bug (see changes in sdb/error.py).

### CODE REVIEW

**Comment by sdimitro** [comment]
cc @grwilson

File: sdb/commands/linux/process.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            .count = (refcount_t){
+                    .refs = (atomic_t){
+                            .counter = (int)50,
+                    },
+            },
+            .level = (unsigned int)0,
+            ...
+    """
+
+    names = ["find_pid"]
```
Comment (ahrens) [inline sdb/commands/linux/process.py]: why "find_pid" rather than plain "pid"?

File: sdb/commands/linux/process.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    @classmethod
+    def _init_parser(cls, name: str) -> argparse.ArgumentParser:
+        parser = super()._init_parser(name)
+        parser.add_argument("pid",
+                            type=int,
+                            nargs="+",
+                            help="numeric PID(s) of process(es)")
```
Comment (ahrens) [inline sdb/commands/linux/process.py]: Seems more usable than `echo 4 | pid` 👍

File: sdb/commands/linux/vfs.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import argparse
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class FGet(sdb.SingleInputCommand):
+    """
+    Return "struct file *" given the FD number for each task_struct piped as input.
```
Comment (ahrens) [inline sdb/commands/linux/vfs.py]: I think if you declare the input_type, we get some benefits like auto casting and type checking.  unless you are intentionally relying on `drgn.helpers.linux.fs.fget()` for that?

File: sdb/commands/linux/process.py (hunk 1 above)
Comment (sdimitro) [reply sdb/commands/linux/process.py]: Changed

File: sdb/commands/linux/vfs.py (hunk 3 above)
Comment (sdimitro) [reply sdb/commands/linux/vfs.py]: No I wasn't. Thanks! input_type added

**Pull request merged**